### PR TITLE
sci-chemistry/erkale: Fix build, deps and tests

### PR DESCRIPTION
**Pull request by Reinis**
- Didn't build because of missing symbols from BLAS
- It requires sci-libs/libint:1
- Tests were failing with <sci-libs/libxc-4.2.0

### CODE REVIEW

File: sci-libs/libxc/libxc-4.2.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	_check_build_dir
+	pushd "${BUILD_DIR}" > /dev/null || die
+	make check || ewarn "Make check failed. See above for details."
+	einfo "emake check done"
+	popd > /dev/null || die
+}
+
+src_install() {
+	autotools-utils_src_install
+
```
Comment (TheChymera) [inline sci-libs/libxc/libxc-4.2.3.ebuild]: empty line

File: sci-libs/libxc/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	<maintainer type="project">
+		<email>sci@gentoo.org</email>
+		<name>Gentoo Science Project</name>
+	</maintainer>
+	<maintainer type="person">
+		<email>Hloupy.Honza@centrum.cz</email>
+		<name>Honza Macháček</name>
+	</maintainer>
+	<longdescription lang="en">
+Libxc is a library of exchange-correlation functionals for density-functional theory.
```
Comment (TheChymera) [inline sci-libs/libxc/metadata.xml]: Please indent

**Comment by Reinis** [comment]
@TheChymera ping

**Comment by TheChymera** [comment]
@Reinis sorry for the delay, if you want the test suite to be accessible on erkale, please add the test flag to the USE variable. Additionally, if you want to restrict the tests on libxc, please add the flag to the RESTRICT variable rather than -flag to USE. You can use `qgrep` to find examples.

**Comment by TheChymera** [comment]
@Reinis are you still interested in pursuing this?

**Comment by Reinis** [comment]
@TheChymera Not really. I'm not using it now and my Gentoo box has reached EOL.

**Comment by TheChymera** [comment]
@Reinis  ok, no worries :) `libxc` is in any case now distributed via ::gentoo rather than ::science. I have submitted a PR for the version bump, including your work as well as a few more fixes, and credited you explicitly in the commit message.

**Comment by TheChymera** [comment]
Here's the PR for libxc btw, feel free to comment: https://github.com/gentoo/gentoo/pull/18454

**Pull request closed without merging**